src/fides/api/ops/service/privacy_request/email_batch_service.py

```diff
@@ -103,7 +105,7 @@ def add_batched_user_preferences_to_emails(
                 pending_email.skipped_privacy_requests.append(privacy_request.id)
 
 
-def send_prepared_emails(
+def send_prepared_consent_emails(
```

Comment:
There's a mismatch now between the types of logs that are created for erasure batch emails and the types of logs created for consent batch emails.  Can we unify here?

Reply:
OK looks like Adrian is adding Execution Logs for both -